framework/util/to_string.h

```diff
@@ -116,7 +116,7 @@ ObjectToString(ToStringFlags toStringFlags, uint32_t& tabCount, uint32_t tabSize
 
 inline void FieldToString(std::stringstream& strStrm,
                           bool               firstField,
-                          const std::string& fieldName,
+                          const char*        fieldName,
```

Comment:
I always like the explicit `const char* const` to tell the compiler we won't be modifying the pointer, but this is fine too.